Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -0,0 +1,438 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
But for the next time what is the process when master has been updated while you've been working on a specific PR ?